src/statsd.ts

```diff
@@ -11,12 +11,14 @@ export interface StatsDOptions {
     statsdClient?: StatsdClient;
 }
 
+const DEBUG_STATSD: boolean = false;
```

Comment:
I left this in here to document the fact that you can control statsD debugging via this param; I know I will never remember this in 3-6 months if I'm trying to debug stats. Maybe adding it to the StatsDOptions type would be a better approach.